playgrounds/react-vite/src/tabs/Collectable.tsx

```diff
@@ -109,79 +111,138 @@ function Actions() {
   };
 
   return (
-    <Card gap="6" justifyContent="center">
+    <Box>
       {!isConnected && (
         <Text variant="large">Connect Wallet to see collectable actions</Text>
       )}
-      <Box gap="3">
-        <Button
-          variant="primary"
-          onClick={() => openMakeOfferModal(hooksProps)}
-          label="Make Offer"
-          disabled={!isConnected}
-        />
-        <Button
-          variant="primary"
-          onClick={() => openCreateListingModal(hooksProps)}
-          label="Buy Item"
-          disabled={!isConnected}
-        />
-      </Box>
-      <Box gap="3">
-        <Button
-          variant="primary"
-          onClick={() => openCreateListingModal(hooksProps)}
-          label="Create Listing"
-          disabled={!isConnected}
-        />
-        <Button
-          variant="primary"
-          onClick={() =>
-            openTransferModal({
-              collectionAddress: context.collectionAddress,
-              chainId: context.chainId,
-              tokenId: context.collectibleId,
-            })
-          }
-          label="Transfer"
-          disabled={!isConnected}
-        />
-      </Box>
-    </Card>
+      <Card gap="6" justifyContent="center">
+        <Box gap="3">
+          <Button
+            variant="primary"
+            onClick={() => openMakeOfferModal(hooksProps)}
+            label="Make Offer"
+            disabled={!isConnected}
+          />
+          <Button
+            variant="primary"
+            onClick={() => openCreateListingModal(hooksProps)}
+            label="Buy Item"
+            disabled={!isConnected}
+          />
+        </Box>
+        <Box gap="3">
+          <Button
+            variant="primary"
+            onClick={() => openCreateListingModal(hooksProps)}
+            label="Create Listing"
+            disabled={!isConnected}
+          />
+          <Button
+            variant="primary"
+            onClick={() =>
+              openTransferModal({
+                collectionAddress: context.collectionAddress,
+                chainId: context.chainId,
+                tokenId: context.collectibleId,
+              })
+            }
+            label="Transfer"
+            disabled={!isConnected}
+          />
+        </Box>
+      </Card>
+    </Box>
   );
 }
 
 function ListingsTable() {
   const { collectionAddress, chainId, collectibleId } = useMarketplace();
+  const [page, setPage] = useState(0);
+
+  const nextPage = () => setPage((prev) => prev + 1);
+  const prevPage = () => setPage((prev) => prev - 1);
```

Comment:
For some reason, we won't able to click go back to previous page if we get the end of the list
<video src="https://github.com/user-attachments/assets/85dac4f1-bec6-465c-bbff-c7e442156339" />